install_pim/docker/installation_docker.rst

```diff
@@ -1,151 +1,144 @@
-- composer cache will be in ``~/.cache/composer``.
 
-Use the Docker images
----------------------
+Non standard directories for Composer and Yarn
+++++++++++++++++++++++++++++++++++++++++++++++
+If you are using non standard directories for Composer and Yarn, you can set the right directories by settting the following environment variable:
+ - Composer: `HOST_COMPOSER_HOME`
+ - Yarn: `HOST_YARN_HOME`
 
-.. note::
 
-   All ``docker-compose`` and ``docker`` commands must be run from the folder containing the *Docker compose* file. For convenience, we provide easy to use commands via `makefile`.
+Creating the PIM project
+------------------------
+Community Edition
+*****************
 
-Currently, our PHP image is not registered on Docker. So you need to build it locally:
+The following command will create a PIM project in the current directory. Please note this directory must be empty.
 
 .. code-block:: bash
 
-	make php-image-dev
+    $ mkdir pim
+    $ cd pim
+    $ docker run -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 \
+        php -d memory_limit=4G /usr/local/bin/composer create-project \
+        akeneo/pim-community-standard /srv/pim "4.0.*@stable"
 
-Make sure you have the latest versions of the images by running:
+.. note::
+    Alternatively, you can download an archive of Akeneo PIM CE containing all the PHP dependencies here:
+    https://download.akeneo.com/pim-community-standard-v4.0-latest.tar.gz
 
-.. code-block:: bash
 
-   $ docker-compose pull --ignore-pull-failures
+Enterprise Edition
+******************
 
-To start your containers, run:
+You need to get a PIM Enterprise Standard archive from the Partners Portal. See  <https://help.akeneo.com/portal/articles/get-akeneo-pim-enterprise-archive.html?utm_source=akeneo-docs&utm_campaign=portal_archive>`_
 
 .. code-block:: bash
 
-   $ make up
-
-To stop the containers, run:
+    $ mkdir pim
+    $ tar -xvjf pim-enterprise-standard-v4.0.tar.gz -C pim
+    $ cd pim
+    $ docker run -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 \
+        php -d memory_limit=4G /usr/local/bin/composer install
 
-.. code-block:: bash
+Launching the PIM in dev mode
+-----------------------------
 
-   $ docker-compose stop
+.. note::
 
-but if you want to completely remove everything (containers, networks and volumes), then run:
+   All `make` commands must be run from the PIM root directory, either created by the archive or from the composer create project above.
 
 .. code-block:: bash
 
-   $ make down
-
-Of course, it does not delete the Akeneo application you cloned on your machine, only the Docker containers. However, it destroys the database and everything it contains.
-
-
-Install and run Akeneo
-----------------------
-
-Configure Akeneo
-****************
+	make dev
```

Comment:
```suggestion
	make pim-dev
```

Reply:
Nope, this is make dev.

Reply:
https://github.com/akeneo/pim-enterprise-dev/blob/master/std-build/Makefile#L63